app/javascript/dashboard/i18n/locale/en/inboxMgmt.json

```diff
@@ -107,6 +107,11 @@
           "PLACEHOLDER": "Please enter your Twilio Account SID",
           "ERROR": "This field is required"
         },
+        "MESSAGING_SERVICE_SID": {
```

Comment:
What's the Chatwoot process for adding new entries in `locale` files?

Reply:
This should be suffice, We push the translation file to crowdin where the community contributors would contribute translations in other languages. 

https://www.chatwoot.com/docs/contributing-guide/translation-guidelines